### ISSUE TITLE
Players cannot join when RTPS really high

### ISSUE THREAD

**Issue by ghost** [MCHPR/MCHPRS#13]
Didn't do much testing on this issue but i found I couldn't login when someone set the RTPS to 35000. Got 'Connection timed out'

**Comment by AggYz** [comment]
Ye, that was my server I repeated it 3 times and whenever the tps is really, really high, Other players weren't able to join it. However I could connect to it using localhost and I then lowered the rtps and you were able to join.

**Comment by Ocawesome101** [comment]
I think that 35000TPS is.... a bit too high to be sane.  Why did you set it that high in the first place?

**Comment by AL1L** [comment]
Maybe the player should have some flag to not send update packets until they properly join the world. Not sure if that'd cause desync issues though. 

Another idea would be to change the RTPS back to 10 when a player is joining for a second?

**Comment by AL1L** [comment]
I think the idea is probably just to not use plot 0,0 for complex redstone, should be a spawn plot.

**Comment by Ocawesome101** [comment]
Another solution could just be to limit the number of packets sent.  Currently, even at just 8000RTPS the server sends nearly 20KB/s (from memory, could be much higher in reality) to the client with just a basic repeater clock.  Only sending the server state every game tick, rather than every time it updates, could greatly help this issue, particularly for those of us with slower internet connections.

Basically just rate-limit the server.

**Comment by StackDoubleFlow** [comment]
Rate limiting was implemented, and this issue should be fixed now. Feel free to reopen if you are still experiencing this issue.
